Resources/views/index.html.twig

```diff
@@ -2,70 +2,82 @@
+            </a>
+        </div>
+    </div>
+
+    <div class="bg-titan-white">
+        <div class="app-outer-container">
+            <div class="">
+                <div class="app-container">
+                    <img class="upper-app" src="/images/app-home.png" alt="Lingo Mac App">
+                </div>
+            </div>
+        </div>
     </div>
 
     <div class="content how-block">
         <h2 class="content-head is-center">How does it works?</h2>
 
-        <div class="pure-g">
-            <div class="how-item pure-u-1 pure-u-md-1-2 pure-u-lg-1-4">
-                <span class="content-subhead">
-                    <i class="fa fa-plug"></i>
-                    Connect to your Slack
-                </span>
-                <p>
-                    Authorize the app to access your Slack team.
-                </p>
+        <section class="step-feature">
+            <img src="/images/step-slack.png">
+            <div class="step-feature-text">
+                <h3>1. Connect to Your Slack</h3>
+                <p>Click on the "Add to Slack" button below then authorize the app to access your Slack team.</p>
             </div>
-            <div class="how-item pure-u-1 pure-u-md-1-2 pure-u-lg-1-4">
-                <span class="content-subhead">
-                    <i class="fa fa-users"></i>
-                    Select users
-                </span>
-                <p>
-                    We show you all your Slack users (<small>excluding the bots, because they don't do nice gifts</small>). Just select which users will take part to your secret santa.
-                </p>
+        </section>
+
+        <section class="step-feature reverse">
+            <img src="/images/step-users.png">
+            <div class="step-feature-text">
+                <h3>2. Select Users</h3>
+                <p>We show you all your Slack users (excluding the bots, because they don't do nice gifts). Just select which users will take part to your secret santa.</p>
             </div>
-            <div class="how-item pure-u-1 pure-u-md-1-2 pure-u-lg-1-4">
-                <span class="content-subhead">
-                    <i class="fa fa-check"></i>
-                    Inform users
-                </span>
-                <p>
-                    After <a href="https://github.com/jolicode/slack-secret-santa/blob/master/src/Rudolph.php">Rudolph</a> shuffled the users, we throw at each participant a private message with his peer name and an optional admin message.
-                </p>
+        </section>
+
+        <section class="step-feature">
+            <img src="/images/step-inform.png">
+            <div class="step-feature-text">
+                <h3>3. Inform Users</h3>
+                <p>After <a href="https://github.com/jolicode/slack-secret-santa/blob/master/src/Rudolph.php">Rudolph</a> shuffled the users, we throw at each participant a private message with his peer name and an optional admin message.</p>
```

Comment:
We could add the Deer emoji (it's brand new): http://emojipedia.org/deer/ 
It will help understanding what Rudolph means.